Bottom: 6d7e1278667b262f6475d7f0930713e0fcaf2164
Top:    84a81eb5208d547308c0551a8daf7274f5f51893
Author: Alexandre Oliva <aoliva@redhat.com>
Date:   2013-08-17 01:25:07 -0300

Thread safety documentation.

for ChangeLog

	* manual/libdl.texi: New.


---

diff --git a/manual/libdl.texi b/manual/libdl.texi
new file mode 100644
index 0000000..e3fe045
--- /dev/null
+++ b/manual/libdl.texi
@@ -0,0 +1,10 @@
+@c FIXME these are undocumented:
+@c dladdr
+@c dladdr1
+@c dlclose
+@c dlerror
+@c dlinfo
+@c dlmopen
+@c dlopen
+@c dlsym
+@c dlvsym
